ports/bsa/vcpkg.json

```
@@ -0,0 +1,20 @@
+{
+  "name": "bsa",
+  "version-semver": "2.0.3",
+  "description": "A C++ library for working with the Bethesda archive file format",
+  "homepage": "https://github.com/Ryan-rsm-McKenzie/bsa",
+  "documentation": "https://ryan-rsm-mckenzie.github.io/bsa/",
+  "supports": "x64 & windows & !uwp",
```

```suggestion
  "supports": "!x86 & !osx & !uwp",
```

This should match the `vcpkg_fail_port_install` line.

---

Now that I've made this change, the CI fails because its GCC version is too old. Should I change the CI baseline?